### DISCUSSION TITLE
Is there a nice way to announce the upcoming deprecation of an attribute in the response body?

### DISCUSSION THREAD

**Discussion by jiyolla** [fastapi/fastapi#6554] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# I am expecting something like this

from pydantic import BaseModel

class User(BaseModel):
    # I want to deprecate this attribute
    user: str
    # And encourage users to this attribute instead
    username: str
    # Is there syntatic support for that?
    # If not, any suggestions?
```

### Description

In the FastAPI documentation, I see there is support for marking deprecation on query parameter and endpoint.

I tried to figure out how to do this by looking into Pydantic and Swagger UI...but it was a little bit hard for me.

So I would like to know if there is support for marking deprecation on single attribute on response body by FastAPI.d

### Operating System

Linux

### FastAPI Version

0.67.0

### Python Version

3.8.5

**Comment by ArcLightSlavik** [comment, marked as answer]
https://github.com/samuelcolvin/pydantic/issues/2255

**Comment by jiyolla** [comment]
> [samuelcolvin/pydantic#2255](https://github.com/samuelcolvin/pydantic/issues/2255)

That's great! Thank you very much
I'll try this when I wake up in the morning and come back to here with the result

**Comment by jiyolla** [comment]
It's working!
Thanks again.
